.github/workflows/linux-snap.yml

```diff
@@ -0,0 +1,43 @@
+name: Linux Snap
+
+on:
+  pull_request:
+    branches:
+    - master
```

Comment:
So this is built on pulls, but then not built on pushes. Which means we'll have tests on what's in pull requests, but not a build for the latest version in master, am I reading that right?
One thing the people on our developer list like to have the ability to do is to quickly test "the current version". While this is a developer mailing list in name, a lot of people aren't really comfortable (or, frankly, able) to build from source - so having a 'current master' Snap would be a great thing for those people - I already create 'current master' AppImages (and macOS and Windows installers).
Would this be a reasonable add, in your opinion?
```
on:
  push:
    branches:
    - master
  pull_request:
    branches:
    - master
```

**Edited to add**:
and of course as I reach the 3rd commit in my review, I see you add that there.
That should really be added here, right?

Reply:
A snap is a single file in the end, correct? So this could be uploaded either as a GitHub artifact or be pushed onto my server. Having it in a 'testing' channel on the store would of course be best (as in - easiest for users to interact with), but I understand that's a bit harder.